src/ghga_transpiler/configs/2.0.yaml

```
@@ -0,0 +1,162 @@
+        case_control_status:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        xref: !!python/object/apply:ghga_transpiler.transformations.to_list []
+        biospecimen_age_at_sampling:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        biospecimen_vital_status_at_sampling:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        biospecimen_isolation:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        biospecimen_storage:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+    sheet_name: Sample
+  - settings:
+      end_column: 23
+      name: experiment_methods
+      transformations:
+        library_type:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        library_selection_methods:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case_list []
+        library_preparation_kit_retail_name:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        primer:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        end_bias:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        target_regions:
+          !!python/object/apply:ghga_transpiler.transformations.to_list []
+        rnaseq_strandedness:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        instrument_model:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        sequencing_layout:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        flow_cell_type:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+        sample_barcode_read:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case []
+    sheet_name: ExperimentMethod
+  - settings:
+      end_column: 6
+      name: experiment_method_supporting_files
+    sheet_name: ExperimentMethodSupportingFile
+  - settings:
+      end_column: 8
+      name: experiments
+    sheet_name: Experiment
+  - settings:
+      end_column: 9
+      name: research_data_files
+      transformations:
+        experiment:
+          !!python/object/apply:ghga_transpiler.transformations.to_list []
+    sheet_name: ResearchDataFile
+  - settings:
+      end_column: 7
+      name: process_data_files
+    sheet_name: ProcessDataFile
+  - settings:
+      end_column: 7
+      name: datasets
+      transformations:
+        types: !!python/object/apply:ghga_transpiler.transformations.to_list []
+    sheet_name: Dataset
+  - settings:
+      end_column: 11
+      name: data_access_policies
+      transformations:
+        data_use_modifier_terms:
+          !!python/object/apply:ghga_transpiler.transformations.to_snake_case_list []
```

```suggestion
        data_use_modifier_terms:
          !!python/object/apply:ghga_transpiler.transformations.to_list []
```
You've used `to_list` on the other `_terms` slots and I think rightfully so, we cannot make any assumptions about how the terms look like in the ontologies.